app/presenters/waste_carriers_engine/registration_presenter.rb

```diff
@@ -0,0 +1,64 @@
+module WasteCarriersEngine
+  class RegistrationPresenter < BasePresenter
+
+    LOCALES_KEY = ".waste_carriers_engine.certificates.registration_pdf"
+
+    def carrier_name
+      return company_name unless business_type == "soleTrader"
+
+      # Based on the logic found in the existing certificate, we simply display
+      # the company name field unless its a sole trader, in which case we take
+      # the details entered into key people. For sole traders there will only be
+      # one, but the list_main_people method still works for finding and
+      # formatting the result found
+      list_main_people
+    end
+
+    # By complex we mean is there a need to display extra detail in the
+    # document. If its a soletrader we display we display an extra section,
+    # which in the case of partners is a list of their names, and for sole
+    # traders its their business name.
+    def complex_organisation_details?
+      return false unless ["soleTrader", "partnership"].include?(business_type)
+      true
+    end
+
+    def complex_organisation_title
```

Comment:
These method names are very similar and a little confusing – what's the difference between a `complex_organisation_title` and a `complex_organisation_name`? Might benefit from some comments or more distinct naming.